python/sglang/srt/server_args.py

```diff
@@ -18,6 +18,7 @@ class ServerArgs:
     context_length: Optional[int] = None
     quantization: Optional[str] = None
     chat_template: Optional[str] = None
+    served_model_name: Optional[Union[str, List[str]]] = None
```

Comment:
1. Why do you allow it to be a list?
2. Could you update it to the CLI args as well? https://github.com/sgl-project/sglang/blob/d360a4ea77f0af1f54e94575173f4fa964dc38fc/python/sglang/srt/server_args.py#L98